src/tests/Deatails.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
```

Comment:
Great job @BezzB 🤝

- [ ] As stated by the previous reviewer, kindly include some unit tests for some pure functions in the Redux code.